pkg/reconciler/domainmapping/reconciler.go

```diff
@@ -94,7 +94,11 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, dm *v1alpha1.DomainMappi
 	}
 
 	// Mapped URL is the metadata.name of the DomainMapping.
-	url := &apis.URL{Scheme: "http", Host: dm.Name}
+	scheme := config.FromContext(ctx).Network.OverrideExternalScheme
+	if scheme == "" {
+		scheme = "http"
+	}
+	url := &apis.URL{Scheme: scheme, Host: dm.Name}
```

Comment:
The docs for the config say [`OverrideExternalScheme overrides the scheme used in external URLs (http by default or https if AutoTLS is enabled) to the set value.` ](https://github.com/knative/networking/blob/c743329dacb980cb302100ceb10877549506f656/pkg/network.go#L269-L270) which, to my reading, implies the setting will override the scheme whether or not AutoTLS is set, but if AutoTLS is set [we swap the scheme back in the `tls()` method](https://github.com/knative/serving/blob/1f5c83f5afe4fe266b8e80850c378309b3e67dd0/pkg/reconciler/domainmapping/reconciler.go#L245). I think we either need to do this logic later, or change the comment to say it only applies when autoTLS is disabled

Reply:
Good point! I'll adjust the comment accordingly.